interface/globals.php

```diff
@@ -209,11 +209,12 @@ function undoMagicQuotes($array, $topLevel=true) {
         }
       }
     }
-    if ($gl_name == 'language_menu_other') {
+    if ($gl_name == 'language_menu_other') {       
       $GLOBALS['language_menu_show'][] = $gl_value;
     }
     else if ($gl_name == 'css_header') {
-      $GLOBALS[$gl_name] = "$rootdir/themes/" . $gl_value;
+        $GLOBALS[$gl_name] = $rootdir.'/themes/'. $gl_value;
+        $GLOBALS['css_theme_name'] = $gl_value;
```

Comment:
In order to not give folks the wrong idea, I'd rec not using a GLOBALS for this value(ie. only used in this script; to support it as a global would mean it would need to be accurate and converted the the rtl css theme name below which will be a pain in the case of a css theme that has rtl in the theme name but doesn't fit the standard rtl naming).